### PR TITLE
2021 03 17 adelacecilia practice coding number one

### PR DESCRIPTION
**Pull request by adelacecilia**

### CODE REVIEW

**Review by M00N89 — COMMENTED** [review]
nice

**Review by RalucaRbc — APPROVED** [review]
💯 Well done Ada

File: web/adelacecilia/html/line-break/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tomato baked eggs recipe</title>
+  </head>
+  <body>
+      <p>Whip up these easy baked eggs<br> with a garlicky tomato sauce and<br> serve with crusty bread  for a lazy weekend brunch,<br> or light lunch, that's sure to satisfy.</p>
+  </body>
+</html>
```
Comment (robertoalx) [inline web/adelacecilia/html/line-break/index.html:9]: Empty line missing, you can fix it in the next PR

**Review by robertoalx — APPROVED** [review]

**Review by biggicode — APPROVED** [review]

**Review by psyhedeliq — APPROVED** [review]
Nice! 🥇

**Review by marianzburlea — APPROVED** [review]
💯

**Pull request merged**